### PR TITLE
Reserve and vendor extension proposal for OPPO

### PR DESCRIPTION
**Pull request by hjiang36**

### CODE REVIEW

**Comment by hjiang36** [comment]
@rpavlik Could you help review this request? Let us know if there is anything we should update. Thanks a lot.

**Comment by rpavlik** [comment]
Nope this looks fine to me, I'll just have to realign it when I go to merge to avoid conflicts. Did you want to reserve more than one or is one all you need?

**Comment by hjiang36** [comment]
Thanks @rpavlik. One is good enough for foreseeable future.
I guess we could submit another request to reserve more when we need them.

Thanks

**Comment by hjiang36** [comment]
@rpavlik Sorry to bother you again. I'm wondering if we also need to create pull request to https://github.com/KhronosGroup/OpenXR-SDK/blob/main/include/openxr/openxr.h or that file would auto-generate changes related to this interaction profile extension?

File: specification/sources/chapters/extensions/oppo/oppo_controller_interaction.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+This interaction profile represents the input sources and haptics on the OPPO Controller.
+
+Supported component paths:
+* On /user/hand/left only:
+** subpathname:/input/x/click
+** subpathname:/input/x/touch
+** subpathname:/input/y/click
+** subpathname:/input/y/touch
+** subpathname:/input/menu/click
+** subpathname:/input/heartrate/value
```
Comment (Wallbraker) [inline specification/sources/chapters/extensions/oppo/oppo_controller_interaction.adoc:37]: The `heartrate` path needs to be decorated with `heartrate_oppo` as per https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#_adding_input_sources_via_extensions

Comment (hjiang36) [reply specification/sources/chapters/extensions/oppo/oppo_controller_interaction.adoc]: @Wallbraker Thanks for reviewing. I updated the path with _oppo decoration per your suggestion in a new commit in this pull request.
Feel free to let me know if there is anything else that needs to be updated.

Thanks

**Comment by rpavlik** [comment]
The headers auto-generate from the XML. I'll pull in the reservation first because there's a review process for the extensions.

**Comment by rpavlik** [comment]
OK, I merged the reservation in #145

**Comment by hjiang36** [comment]
Thanks @rpavlik for merge the reserve commit. 
I just changed the base of the pull request to staging branch and resolved conflict.
I used the newly assigned extension ID as well.

If you or @Wallbraker could help review it again, it would be greatly appreciated.

**Pull request closed without merging**